File: official/docs/node/v6/smartrate/retrieve-time-in-transit-statistics.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const shipment = await client.Shipment.retrieve('shp_...');

  const smartRates = shipment.getSmartrates(shipment.id);

  console.log(smartRates);
})();

File: official/docs/csharp/v6/smartrate/retrieve-time-in-transit-statistics.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            List<EasyPost.Models.API.SmartRate> smartRates = await client.Shipment.GetSmartRates("shp_...");

            Console.WriteLine(JsonConvert.SerializeObject(smartRates, Formatting.Indented));
        }
    }
}
